modules/project/README.md

```diff
@@ -252,6 +252,29 @@ module "service-project" {
 # tftest modules=2 resources=8 inventory=shared-vpc.yaml
 ```
 
+The module allows also granting necessary permissions in host project to service identities by specifying which services will be used in service project in `grant_iam_for_services`.
+```hcl
+module "host-project" {
+  source = "./fabric/modules/project"
+  name   = "my-host-project"
+  shared_vpc_host_config = {
+    enabled = true
+  }
+}
+
+module "service-project" {
+  source = "./fabric/modules/project"
+  name   = "my-service-project"
+  shared_vpc_service_config = {
+    host_project = module.host-project.project_id
+    grant_iam_for_services = [
+      "container.googleapis.com",
```

Comment:
I would skip `.googleapis.com`.

Reply:
I kind of like the googleapis suffix as it kind of conveys that the field only makes sense if you're using that API (i.e. you wouldn't put container here if you don't enable the container API)

Reply:
I see your point, it's fine it's not like typing it 2-3 times would be a problem